sdks/python/apache_beam/io/lyft/s3_and_kinesis.py

```diff
@@ -0,0 +1,96 @@
+import json
+import logging
+import random
+import string
+from collections import namedtuple
+
+from apache_beam import PTransform
+from apache_beam.pvalue import PBegin
+from apache_beam.pvalue import PCollection
+from apache_beam.transforms.core import Windowing
+from apache_beam.transforms.window import GlobalWindows
+
+Event = namedtuple('Event', 'name lateness_in_sec lookback_days')
+KinesisConfig = namedtuple('KinesisConfig', 'name properties parallelism stream_start_mode')
```

Comment:
How about converging onto a single config file which honors the existing kinesis config properties as defined @ https://github.com/lyft/beam/blob/STRMCMP-996/sdks/python/apache_beam/io/lyft/kinesis.py#L41